src/components/DropdownMenu/DropdownMenu.vue

```diff
@@ -2,12 +2,14 @@
     <ul class="dropdown-menu">
         <slot name="prepend" />
 
-        <DropdownMenuItem
-            v-for="(item, index) of items"
-            :key="`item-${index}`"
-            v-bind="item"
-            @click="clickItem(item)"
-        />
+        <slot name="items">
```

Comment:
```suggestion
        <slot name="items" :click="clickItem">
```
I think we should pass the click event also so we can trigger click from custom dropdown item